sdk/search/Azure.Search.Documents/src/Batching/Publisher{T}.cs

```diff
@@ -134,9 +134,16 @@ public int IndexingActionsCount
         /// The number of bytes to use when tuning the behavior of the
         /// publisher.
         /// </param>
-        /// <param name="retryCount">
+        /// <param name="maxRetries">
         /// The number of times to retry a failed document.
         /// </param>
+        /// <param name="retryDelay">
+        /// The initial retry delay on which to base calculations for a
+        /// backoff-based approach.
```

Comment:
I'd perhaps use more plain language here, like "The initial retry delay. Subsequent retries may take longer up to the <paramref name="maxRetryDelay"/>."

Reply:
Fixed

Reply:
Fixed